SemDiff.Core/GitHub.cs

```diff
@@ -173,6 +218,23 @@ private static string ParseNextLink(IEnumerable<string> links)
             {
                 return null;
             }
+            if (CurrentSaved != null)
+            {
+                var removePRs = CurrentSaved;
+                foreach (var pr in pullRequests)
+                {
+                    foreach (var prRemove in removePRs)
+                    {
+                        if (pr.Number == prRemove.Number)
+                        {
+                            removePRs.Remove(prRemove);
```

Comment:
I believe(based on how visual studio highlights things) that it doesn't throw an error because the break escapes the inner foreach loop.